sdks/python/apache_beam/transforms/core.py

```diff
@@ -622,6 +638,16 @@ class DoFn(WithTypeHints, HasDisplayData, urns.RunnerApiFn):
   def from_callable(fn):
     return CallableWrapperDoFn(fn)
 
+  @staticmethod
+  def unbounded_per_element():
```

Comment:
This should also control whether the PCollection is unbounded or not during expand.

Reply:
I created a JIRA to track this: https://issues.apache.org/jira/browse/BEAM-10380. I want to have a separate PR to do this since it is out of the scope of this PR.

Reply:
sg